rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -97,6 +97,22 @@ object Runtime {
     val REG_PUBLIC_REGISTER_INSERT_CALLBACK: Long = 19L
   }
 
+  def byteName(b: Byte): Par = GPrivate(ByteString.copyFrom(Array[Byte](b)))
+
+  object FixedChannels {
+    val STDOUT: Par            = byteName(0)
+    val STDOUT_ACK: Par        = byteName(1)
+    val STDERR: Par            = byteName(2)
```

Comment:
I very much like this refactoring, but would even love it if it were a separate commit :) Just FYI, the commit and the PR is small enough for this to not matter really.